packages/core/scripts/collateral-umip/1_Propose.js

```diff
@@ -20,40 +41,74 @@ async function runExport() {
     throw "Must provide --fee and --collateral";
   }
 
-  // The proposal will first add a final fee for the currency.
-  const store = await Store.deployed();
-  const addFinalFeeToStoreTx = store.contract.methods
-    .setFinalFee(argv.collateral, { rawValue: web3.utils.toWei(argv.fee) })
-    .encodeABI();
-  console.log("addFinalFeeToStoreTx", addFinalFeeToStoreTx);
+  const collaterals = _.castArray(argv.collateral);
+  const fees = _.castArray(argv.fee);
+  const decimals = argv.decimals && _.castArray(argv.decimals);
 
-  // The proposal will then add the currency to the whitelist.
-  const whitelist = await AddressWhitelist.deployed();
-  const addCollateralToWhitelistTx = whitelist.contract.methods.addToWhitelist(argv.collateral).encodeABI();
-  console.log("addCollateralToWhitelistTx", addCollateralToWhitelistTx);
+  if (collaterals.length !== fees.length || (decimals && decimals.length !== collaterals.length)) {
+    throw "Must provide the same number of elements to --fee, --collateral, and --decimals (optional)";
+  }
 
-  // Send the proposal
-  const governor = await Governor.deployed();
-  await governor.propose(
-    [
-      {
-        to: store.address,
-        value: 0,
-        data: addFinalFeeToStoreTx
-      },
-      {
+  const argObjects = _.zipWith(collaterals, fees, decimals, (collateral, fee, numDecimalsArg) => {
+    return { collateral, fee, numDecimalsArg };
```

Comment:
this is cool, so it looks like you added ability to pass multiple collaterals in. could you show how to do this in the comment for this file? i think i could probably guess the right way but it prob should be documented

Reply:
This should already be up there -- notice that the collateral and fees args are provided twice in the example. Is that explanatory enough or is there more context I could add?

Reply:
Added a bit more context about how to apply the args below the example.